src/processout/processout.ts

```diff
@@ -864,6 +908,35 @@ module ProcessOut {
             }.bind(this);
         }
 
+        protected buildHandleTokenInvoiceAction(
+          invoiceID: string,
+          gatewayConf: any,
+          customerTokenId?: string,
+        ): (
+          tokenized: (token: string) => void,
+          tokenError: (err: Exception) => void
+        ) => ActionHandler {
```

Comment:
I was thinking we could define a separate type for this function just to not repeat it few times, but I can see it's already repeated across this file, so not sure 😄

Reply:
Yeah, we could. But I feel I made enough changes already. Don't want to end up refactoring the entire codebase. 😅